### PR TITLE
Add link to join Slack to the bottom of event details

### PR DESCRIPTION
**Pull request by JakeSidSmith**
Screenshot 2023-11-27 at 14 58 23

### CODE REVIEW

**Review by yanneves — CHANGES_REQUESTED** [review]

**Comment by yanneves** [comment]
@JakeSidSmith I did a little refactor to deduplicate the links, see if that looks right to you

**Review by yanneves — APPROVED** [review]

**Comment by JakeSidSmith** [comment]
Note: link is not present in nested partials. Needs some investigation.

File: _includes/partials/nextEvent.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <section class="module event-detail">
   <h2 class="module-heading">{{{ iconmonstr 'calendar-tick' }}}Next Event</h2>
   {{#each soonest}}
-    {{> eventDetail }}
+    {{> eventDetail site=../site }}
```
Comment (yanneves) [inline _includes/partials/nextEvent.html:4]: @JakeSidSmith this was the issue!

the nested partial in `{{> nextEvent}}` renders `{{> eventDetail}}` scoped to this `{{#each soonest}}` loop context, so could no longer read from the global scope (where `site` lives)

fixed by hoisting a reference to `site` here `site=../site`

relative paths to read variables just reminds me how far we've progressed in this post-JSX era 😂

**Comment by yanneves** [comment]
> Note: link is not present in nested partials. Needs some investigation.

Now fixed, this should be good to go 🚀

File: _includes/partials/nextEvent.html (hunk 1 above)
Comment (JakeSidSmith) [reply _includes/partials/nextEvent.html:4]: Loops don't receive the outer context/scope? 🤨 That doesn't seem right.

Screenshot 2024-03-14 at 16 37 55


I'm happy to accept this as the fix though.

**Pull request merged**